sql-join/big-spenders.sql

```diff
@@ -0,0 +1,5 @@
+select *
+from "payments"
+join "customers" using ("customerId")
+order by "payments"."amount" DESC
+limit 10;
```

Comment:
Only select the fields included in the spec:

---

![image](https://user-images.githubusercontent.com/16143771/179895931-a8cd48c9-2f83-4762-84c3-d4ad2a29c95b.png)

Reply:
@TimelikeClosure updated!
![image](https://user-images.githubusercontent.com/25317196/179896904-d64f763d-a19d-40e9-bb68-4b07869f88b0.png)